app/frontend/courses/report/components/CoursesReport__Overview.res

```diff
@@ -213,19 +260,69 @@ let levelProgressBar = (levelId, levels, levelsCompleted) => {
 }
 
 @react.component
-let make = (~overviewData, ~levels, ~coaches) =>
+let make = (~overviewData, ~coaches) =>
   <div className="max-w-3xl mx-auto">
     {switch overviewData {
     | OverviewData.Loaded(overview) =>
       <div className="flex flex-col">
         <div className="w-full">
-          {levelProgressBar(
-            overview |> StudentOverview.levelId,
-            levels,
-            overview |> StudentOverview.completedLevelIds,
-          )}
-          <div className="mb-8">
-            <h6 className="font-semibold"> {t("targets_overview") |> str} </h6>
+          <div>
+            <div className="flex justify-between mt-8">
+              <p className="text-sm font-semibold">
+                <span> {t("milestone_status_title")->str} </span>
+                <HelpIcon className="ml-2"> {t("milestone_status_help")->str} </HelpIcon>
+              </p>
+              {milestoneTargetsCompletionStatus(overview)}
+            </div>
+            <div className="grid gap-2 mt-2">
+              {ArrayUtils.copyAndSort(
+                (a, b) =>
+                  a->CoursesReport__MilestoneTargetCompletionStatus.milestoneNumber -
+                    b->CoursesReport__MilestoneTargetCompletionStatus.milestoneNumber,
+                StudentOverview.milestoneTargetsCompletionStatus(overview),
+              )
+              ->Js.Array2.map(data => {
+                <a
+                  href={"/targets/" ++ CoursesReport__MilestoneTargetCompletionStatus.id(data)}
+                  className="flex col-span-1 items-center justify-between p-2 rounded-md border bg-gray-100 hover:bg-primary-100 hover:border-primary-500 hover:text-primary-500 transition">
+                  <div className="flex items-center">
+                    <p className="text-sm font-semibold mr-2">
+                      {("M" ++
+                      string_of_int(
+                        CoursesReport__MilestoneTargetCompletionStatus.milestoneNumber(data),
+                      ))->str}
+                    </p>
+                    <div
+                      className="flex-1 text-sm truncate max-w-[20ch] sm:max-w-[30ch] md:max-w-[28ch]">
```

Comment:
@vinutv 

I see a lot of space between the milestone text and status on larger screens. How about adding another breakpoint `lg` 🤔?

![image](https://github.com/pupilfirst/pupilfirst/assets/53794102/efdfda49-7cc7-4530-bd1c-f286bee9fd12)

Reply:
@yash-learner  Updated this. https://github.com/pupilfirst/pupilfirst/commit/3918843634ddb9ca3a35e60eee5f9c91313a14aa